integrations/docker/images/chip-build-qpg6100/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+# ==================================================
+# Needed tooling
+# ==================================================
+
+# GNU ARM Embedded toolchain, cross compiler for various platform builds
```

Comment:
current pigweed environment should install a compatible arm gcc.

Could we use that instead of having a separate checkout? I would imagine we want to compile chip across all arm platforms with a consistent compiler if possible.